editor/export/editor_export_platform.cpp

```diff
@@ -1230,7 +1230,9 @@ Error EditorExportPlatform::export_project_files(const Ref<EditorExportPreset> &
 
 				// Erase useless sections.
 				config->erase_section("deps");
```

Comment:
Should this also have a check just in case?

Reply:
I think "deps" always has elements in it since it stores the path of the source file and the imported file. So we shouldn't need to check it